pydantic_ai_slim/pydantic_ai/messages.py

```diff
@@ -949,7 +972,13 @@ def model_response(self) -> str:
             description = (
                 f'{len(self.content)} validation error{"s" if plural else ""}:\n```json\n{json_errors.decode()}\n```'
             )
-        return f'{description}\n\nFix the errors and try again.'
+
+        if self.retry_message is None:
```

Comment:
The "Validation feedback" bit above needs to be customizable as well!

Reply:
Oh yep, that makes sense. Missed that

Reply:
So I added a description_template, which I am restricting to be a Callable so that the users are aware content is available to be used while generating it. It isn't pretty but useful.